homeassistant/components/homewizard/sensor.py

```diff
@@ -74,7 +74,11 @@ class HomeWizardSensorEntityDescription(
         key="active_tariff",
         name="Active tariff",
         icon="mdi:calendar-clock",
-        value_fn=lambda data: data.active_tariff,
+        value_fn=lambda data: str(data.active_tariff)
+        if data.active_tariff is not None
+        else None,
```

Comment:
Multi-line lambdas should be avoided.
Maybe you can create a `_ensure_string` method that `return None if value is None else str(value)`
```suggestion
        value_fn=lambda data: _ensure_string(data.active_tariff),
```

Reply:
Single line lambda that fits:

```suggestion
        value_fn=lambda data: (
            str(data.active_tariff) if data.active_tariff is not None else None
        ),
```

Reply:
Or an shorter variant:

```suggestion
        value_fn=lambda data: (
            None if data.active_tariff is None else str(data.active_tariff)
        ),
```

Reply:
Thanks, was looking for something like this!